package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "lasso-babel-transform",
-  "version": "1.0.1",
+  "version": "1.2.0",
```

Comment:
Please revert this change. I will apply this after merge so we can maintain git tags.